### ISSUE TITLE
Anorectal abscess

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6068]
**Preferred term label:**
Anorectal abscess

**Synonyms**
Anal abscess

**Definition (free text, please give PubMed ID)**
A collection of pus near the anus.
PMID: 29939672

**Parent term (use hpo.jax.org/app)**
Abscess
-> "Anorectal abscess" as the parent term of "Perianal abscess"

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 107100

**Comment by pnrobinson** [comment]
added as HP:0033150
